gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -215,61 +182,15 @@ public void orchestrate(Spec spec, Properties jobProps, long triggerTimestampMil
-        DagActionStore.LeaseParams
-            leaseObject = new DagActionStore.LeaseParams(launchDagAction, isReminderEvent,
-            triggerTimestampMillis);
-        // `flowSpec.isScheduled()` ==> adopt consensus `flowExecutionId` as clock drift safeguard, yet w/o disrupting API-layer's ad hoc ID assignment
-        flowLaunchHandler.get().handleFlowLaunchTriggerEvent(jobProps, leaseObject, flowSpec.isScheduled());
-        _log.info("Multi-active scheduler finished handling trigger event: [{}, is: {}, triggerEventTimestamp: {}]",
-            launchDagAction, isReminderEvent ? "reminder" : "original", triggerTimestampMillis);
-      } else {
-        try {
-          TimingEvent flowCompilationTimer =
-              new TimingEvent(this.eventSubmitter, TimingEvent.FlowTimings.FLOW_COMPILED);
-          Map<String, String> flowMetadata = TimingEventUtils.getFlowMetadata(flowSpec);
-          Optional<Dag<JobExecutionPlan>> compiledDagOptional =
-              this.flowCompilationValidationHelper.validateAndHandleConcurrentExecution(flowConfig, flowSpec, flowGroup,
-                  flowName, flowMetadata);
-
-          if (!compiledDagOptional.isPresent()) {
-            Instrumented.markMeter(this.flowOrchestrationFailedMeter);
-            return;
-          }
-          Dag<JobExecutionPlan> compiledDag = compiledDagOptional.get();
-          if (compiledDag.isEmpty()) {
-            FlowCompilationValidationHelper.populateFlowCompilationFailedEventMessage(eventSubmitter, flowSpec,
-                flowMetadata);
-            Instrumented.markMeter(this.flowOrchestrationFailedMeter);
-            sharedFlowMetricsSingleton.conditionallyUpdateFlowGaugeSpecState(spec,
-                SharedFlowMetricsSingleton.CompiledState.FAILED);
-            _log.warn("Cannot determine an executor to run on for Spec: " + spec);
-            return;
-          }
-          sharedFlowMetricsSingleton.conditionallyUpdateFlowGaugeSpecState(spec,
-              SharedFlowMetricsSingleton.CompiledState.SUCCESSFUL);
-
-          FlowCompilationValidationHelper.addFlowExecutionIdIfAbsent(flowMetadata, compiledDag);
-          flowCompilationTimer.stop(flowMetadata);
-
-          // Depending on if DagManager is present, handle execution
-          submitFlowToDagManager(flowSpec, compiledDag);
-        } finally {
-          /* Remove adhoc flow spec from the flow catalog, regardless of whether the flow was successfully validated
-          and permitted to exec (concurrently)
-           */
-          this.dagManager.removeFlowSpecIfAdhoc(flowSpec);
-        }
-      }
+      DagActionStore.DagAction launchDagAction = DagActionStore.DagAction.forFlow(flowGroup, flowName,
+          FlowUtils.getOrCreateFlowExecutionId(flowSpec), DagActionStore.DagActionType.LAUNCH);
+      DagActionStore.LeaseParams
+          leaseObject = new DagActionStore.LeaseParams(launchDagAction, isReminderEvent,
+          triggerTimestampMillis);
```

Comment:
nit: don't leave the type on a line by itself, but also put the `leaseObject` variable name there too